L.Control.ZoomBox.js

```diff
@@ -4,15 +4,26 @@ L.Control.ZoomBox = L.Control.extend({
     includes: L.Evented ? L.Evented.prototype : L.Mixin.Events,
     options: {
         position: 'topleft',
-        className: 'leaflet-zoom-box-icon',
+        forceSeparateButton: true,
```

Comment:
@Azmisov this is an interesting idea. What about flipping this so that it is a bit more explicitly opt-in: `isZoomControl: false`, `joinZoomControl: false`, `addToZoomControl: false`, or similar?

Reply:
Okay, I changed it to `addToZoomControl`, default being false